core-bundle/src/Resources/contao/templates/elements/ce_youtube.html5

```diff
@@ -17,12 +17,13 @@
             iframe.src = this.href;
             iframe.width = '<?= $this->width ?>';
             iframe.height = '<?= $this->height ?>';
+            iframe.setAttribute('allow', 'autoplay; encrypted-media; picture-in-picture; fullscreen');
             iframe.setAttribute('allowfullscreen', '');
             this.parentNode.replaceChild(iframe, this);
           });
         </script>
       <?php else: ?>
-        <iframe<?= $this->size ?> src="<?= $this->src ?>" allowfullscreen></iframe>
+        <iframe<?= $this->size ?> src="<?= $this->src ?>" allow="autoplay; encrypted-media; picture-in-picture; fullscreen" allowfullscreen></iframe>
```

Comment:
Is this change even right? Because according to https://developers.google.com/web/updates/2017/09/autoplay-policy-changes#new-behaviors, _autoplay with sound will only work if the user has interacted with the domain (click, tap, etc.)_. This is not the case in the `else` condition where there is no splash image.

Reply:
IMO yes. Other browsers might act differently and AFAIK you can allow specific domains to always autoplay videos for certain domains.